task_config_template.cfg

```diff
@@ -341,6 +341,11 @@ patterns = ['images/sec: (\d+\.\d+)']
 metrics = ['Speed(Images/sec)']
 compute_method = ['average']
 
+[pytorch_resnet50_synthetic_p3_16xlg_runner]
+patterns = ['BatchSpeed (\d+\.\d+|\d+)', 'Time cost=(\d+\.\d+|\d+)']
```

Comment:
this should have both BatchSpeed and EpochSpeed

```
[resnet50_synthetic_p3_16xlg_runner]
patterns = ['Speed: (\d+\.\d+|\d+) samples/sec', 'BatchSpeed (\d+\.\d+|\d+)', 'Time cost=(\d+\.\d+|\d+)']
metrics = ['speed(samples/sec)', 'batchspeed(samples/sec)','Time']
compute_method = ['average', 'average', 'last']
```

let me know if it makes sense